### PR TITLE
FPC teaser landing page

### PR DESCRIPTION
**Pull request by maartenweyns**

### CODE REVIEW

File: static/fpc/teaser/index.html [hunk 1]
```diff
@@ -10,8 +10,8 @@
 					<img src="../fpc-logo.png" class="logo">
 				</td>
 				<td style="width: 70%">
-					<h1>1. Glass Reflection</h1>
-					<h4>FPC 2023 First Teaser Puzzle</h4>
+					<h1>Hi there!</h1>
+					<h4>Welcome to the FPC 2024 teaser</h4>
 					<!-- Source: FPC 2022, https://chipcie.wisv.ch/archive/fpc/, CC BY-SA 4.0 -->
```
Comment (mpsijm) [inline static/fpc/teaser/index.html:15]: The `Source` can be removed from the new landing page :slightly_smiling_face:

File: static/fpc/teaser/index.html [hunk 2]
```diff
@@ -21,60 +21,31 @@ <h4>FPC 2023 First Teaser Puzzle</h4>
...
-							<tr>
-								<td class="code">sskkiilllleesssnneesss</td>
-								<td class="code">skilllessness</td>
-							</tr>
-							<tr>
-								<td class="code">bbaalllooonnnoookkkeeeppeerr </td>
-								<td class="code">balloonnookkeeper</td>
-							</tr>
-						</table>
+						<h2>Introduction sessions</h2>
```
Comment (mpsijm) [inline static/fpc/teaser/index.html:31]:
```suggestion
						<h2>Introduction session</h2>
```

File: static/fpc/style.css [hunk 3]
```diff
@@ -112,5 +110,12 @@ table.title {
 }
 
 body {
-	background-color: black;
+    background-color: black;
```
Comment (mpsijm) [inline static/fpc/style.css:113]: I know spaces are better than tabs, but I prefer consistency :innocent: Can you change the indentation back to tabs? (or change everything to spaces, I guess, but that's a bit more work and pollutes the diff :joy: )

**Review by mpsijm — APPROVED** [review]
I think the teaser is ready to go, do you have a reason for keeping the draft status other than that we don't know the exact location/time yet for the intro session? In my opinion, we can merge it like this and add these details later :slightly_smiling_face:

**Comment by maartenweyns** [comment]
> I think the teaser is ready to go, do you have a reason for keeping the draft status other than that we don't know the exact location/time yet for the intro session? In my opinion, we can merge it like this and add these details later 🙂

Yeah, I was able to confirm the date with Verwoerd on Slack. Just a link to the page on events, but that can indeed be added later. I'll merge this PR.

**Comment by maartenweyns** [comment]
Force push 😮

**Comment by mpsijm** [comment]
Force pushing development branches is fine, you can't really get around that when you rebase :joy:

**Pull request merged**